pgvectorscale/sql/vectorscale--0.4.0--0.5.0.sql

```diff
@@ -0,0 +1,89 @@
+/* <begin connected objects> */
```

Comment:
we usually add this file during release and not in PRs. Why was this needed now? for testing?